src/moves/FreeScroll.ts

```diff
@@ -45,7 +47,7 @@ class FreeScroll extends Snap {
       return {
         panel: estimatedPanel,
         destPos,
-        duration: viewport.options.duration,
+        duration: Math.max(axesEvent.duration, viewport.options.duration),
```

Comment:
@daybrush I think duration should follow only axesEvent.duration in freeScroll mode because viewport.option.duration is only applied when moving between panels.